generator/csv/fields.csv

```diff
@@ -876,10 +876,10 @@ SavePartyLocation,unboarding,f,Boolean,0x68,False,0,0,
 SavePartyLocation,preboard_move_speed,f,Int32,0x69,4,0,0,Move speed before the party boarded the vehicle
 SavePartyLocation,menu_calling,f,Boolean,0x6C,False,0,0,Flag which briefly is true if the player presses ESC. At the right place in handling each frame's activities for the player; the code checks whether this flag is set and calls the menu; however there are several conditions which would cancel this flag and instead process another higher-priority action; such as when an encounter takes place during the same frame.
 SavePartyLocation,pan_state,f,Int32,0x6f,1,0,0,
-SavePartyLocation,pan_current_x,f,Int32,0x70,2304,0,0,int
-SavePartyLocation,pan_current_y,f,Int32,0x71,1792,0,0,?
-SavePartyLocation,pan_finish_x,f,Int32,0x72,2304,0,0,?
-SavePartyLocation,pan_finish_y,f,Int32,0x73,1792,0,0,?
+SavePartyLocation,pan_current_x,f,Int32,0x70,0,0,0,int
+SavePartyLocation,pan_current_y,f,Int32,0x71,0,0,0,?
+SavePartyLocation,pan_finish_x,f,Int32,0x72,0,0,0,?
+SavePartyLocation,pan_finish_y,f,Int32,0x73,0,0,0,?
```

Comment:
iirc I directly ripped the save defaults out of RPG_RT.exe, checking this later

Reply:
Those might be the correct defaults, but in Player we set them to 0. Then they get written back out to the LSD.

I don't know what values these are in `RPG_RT.exe`, but they are defaulted and not written out.

It's possible your defaults were correct and we are interpreting them differently in Player.

Reply:
The defaults are indeed 9*256 and 7*256 (center - as the panning is stored in 1/16 pixels)

Reply:
Thanks, I'll take a look at how this is used in Player and come up with a better solution here.

Reply:
This change was removed.